integration/embed_test.go

```diff
@@ -115,3 +117,34 @@ func setupEmbedCfg(cfg *embed.Config, curls []url.URL, purls []url.URL) {
 	}
 	cfg.InitialCluster = cfg.InitialCluster[1:]
 }
+
+func TestErrOnEmbedClose(t *testing.T) {
```

Comment:
I was not able to reproduce error on close with the above suggestion. Were you able too?